Nos/Models/ReportCategory.swift

```diff
@@ -57,7 +57,13 @@ enum ReportCategoryType {
         code: "HC",
         nip56Code: .other
     )
-    
+
+    static let harassment = ReportCategory(
+        name: .moderation.harassment,
+        code: "HR",
```

Comment:
"HA" seemed wrong for this, so I went with "HR" instead.

Reply:
This is not very explicit but we are trying to make the categories sync with what [this draft proposes](https://github.com/rabble/nips/blob/724e05e762a634e501bdcf6cbefaa91f99b1903b/69.md) and what [cleanstr reports](https://github.com/planetary-social/cleanstr/blob/main/src/lib/openAICategories.js).

I'm ok with the idea of changing this standard but if we do it I think we'd need to be sure that cleanstr uses the same vocabulary and we should probably have a proposed nip for these changes, @rabble wdyt?

Reply:
@dcadenas Oh, well then I'm fine with changing this. Should it be "IL-har", based on those?

Reply:
yes, I think that one is the best match